sycl/include/sycl/ext/intel/experimental/esimd/math.hpp

```diff
@@ -1592,6 +1592,37 @@ asin(T src0, int flag = saturation_off) {
   return Result[0];
 }
 
+// log
+
+template <int SZ>
+ESIMD_NODEBUG ESIMD_INLINE simd<float, SZ> log(simd<float, SZ> src0,
```

Comment:
please add a doc comment that this is s/w-emulated version based on h/w supported log2